perfkitbenchmarker/providers/alicloud/ali_virtual_machine.py

```diff
@@ -91,13 +88,14 @@ def __init__(self, vm_spec):
       vm_spec: virtual_machine.BaseVirtualMachineSpec object of the VM.
     """
     super(AliVirtualMachine, self).__init__(vm_spec)
-    self.image = self.image or DEFAULT_IMAGE
+    self.image = FLAGS.image
```

Comment:
Seems like we should still support a default image in the derived classes?

Reply:
I removed the `DEFAULT_IMAGE` because it did no longer exist. Adding default images to the derived classes will require maintenance, as the provider changes them. Instead, I used `IMAGE_NAME_FILTER` to find the right image in a more sustainable fashion. Please let me know what you think.